content/guides/analog-discovery/_index.md

```diff
@@ -0,0 +1,886 @@
+
+ * load the project, by calling *enable()* on the project in the shuttle; and 
+ * set the input bits so that the ring oscillator is on, using the *input_byte* to set all the bits in one go
+ 
+```
+
+>>> tt.shuttle.tt_um_tt05_analog_test.enable()
+ttboard.project_mux: Enable design tt_um_tt05_analog_test 
+>>> tt.input_byte = 0b1100000
+
+```
+ 
+{{< figure src="images/7.png" title="REPL interaction" >}}
+
+The ring oscillator should now be running, output on the pin connected to channel 1 and visible in the scope (click *Run*--the "play" icon--if you can't see it, or check the connections).
+ 
+{{< figure src="images/8.png" title="OSC on scope" >}}
+
+The scroll wheel to zoom in and drag the trigger level (yellow arrow on right side) to get something stable.
+Use the range on the right to get a better look by stretching it vertically, and drag the offest arrow on the left to position it.
+
+
+{{< figure src="images/11.png" title="OSC zoomed in and stable" >}}
+
+Disabling channel 2 for now, using the checkbox, clears up the display, and the measuremnt tools may be use to look at width (time intervals), height (voltage) and pulse length.  These are easy to use with the mouse: just click on a location to begin a measurement, and a second location if it involves a delta.  
+
+Clicking again will start a new measurement, double-clicking clears away the tool.
+
+
+{{< figure src="images/12.png" title="OSC pulse measurement" >}}
+
+The pulses do seem to go from rail to rail but, using the pulse measurement, the period can be seen to be a little over 400kHz, which is about 100kHz under the expected value.
+
+There are numerous tools available on the scope.  For automated measurements of many types, click "Measurements".  This opens another pane.
+
+{{< figure src="images/14.png" title="Measurements pane" >}}
+
+Things can get crowded, so two interesting things to note are:
+
+ * the arrow above the channel pane allows it to be set to auto-hide; and
+ * grabbing a pane by it's title, "Measurements" here, allows it to be dragged to other positions.
+ 
+
+{{< figure src="images/16.png" title="Measurements pane reposition" >}}
+ 
+Using the "+ Add" button in Measurements, selecting 
+ Defined Measurements -> Channel 1 -> Horizontal -> Frequency 
+Provides a real-time measurement for this value.  
+
+
+{{< figure src="images/17.png" title="Measurements selection" >}}
+
+
+Many other measurements are available. 
+
+## DAC 
+
+To look at the DAC function, channel 2 can be re-enabled using the checkbox and scaled an positioned just like C1.
+
+{{< figure src="images/18.png" title="Measurements made" >}}
+
+
+There's a good deal of cross-talk, likely owing to the dangly wires used to probe the analog pins, so the ring oscillator is disabled while the DAC is enabled, by setting the in3 bit and clearing the rest, in the REPL:
+
+```
+>>> tt.input_byte = 0
+>>> tt.in3(1)
```

Comment:
stick to one style of setting the pins? sometimes you use input_byte, sometimes in3 - a bit confusing